draft-ietf-aipref-vocab.md

```diff
@@ -203,11 +203,144 @@ If arrangements, such as contracts exist that explicitly permit the use of that
+|:-----------------------|:------|:----------|
+| Text and Data Mining   | tdm   | {{tdm}}   |
+| AI Training            | ai    | {{ai}}    |
+| Generative AI Training | genai | {{genai}} |
+{: #t-category-labels title="Mappings for Categories"}
+
+Any mapping for a new usage category can only use
+lowercase latin characters (a-z), digits (0-9), "_", "-", ".", or "*".
+These are encoded using the mappings in {{!ASCII=RFC0020}}.
+
+
+## Preference Labels
+
+The abstract model used has two options for preferences associated with each category:
+allow and disallow.
+These are mapped to single byte Tokens ({{Section 3.3.4 of !FIELDS}})
+of "y" and "n", respectively (without the quotes).
+
+
+## Text Encoding
+
+Structured Fields {{!FIELDS=RFC9651}} describes a byte-level encoding of information,
+not a text encoding.
+This makes this format suitable for inclusion in any protocol or format that carries bytes.
+
+Some formats are defined in terms of strings rather than bytes.
+These formats might need to decode the bytes of this format to obtain a string.
+As the syntax is limited to ASCII {{?ASCII=RFC0020}},
+an ASCII decoder or UTF-8 decoder {{?UTF8=RFC3629}} can be used.
+This results in the strings that this document uses.
+
+Processing (see {{processing}}) requires a sequence of bytes,
+so any format that uses strings needs to encode strings first.
+Again, this process can use ASCII or UTF-8.
+
+
+## Syntax Extensions
+
+There are two ways by which this syntax might be extended:
+the addition of new labels and the addition of parameters.
+
+New labels might be defined to correspond to new usage categories.
+{{vocab-extension}} addresses the considerations for defining new categories.
+New labels might also be defined for other types of extension
+that do not assign a preference to a usage category.
+In either case, when processing a parsed Dictionary to obtain preferences,
+any unknown labels MUST be ignored.
+
+The Dictionary syntax ({{Section 3.2 of !FIELDS}}) can associate parameters
+with each key-value pair.
+This document does not define any semantics for any parameters that might be included.
+When processing a parsed Dictionary to obtain preferences,
+any unknown parameters MUST be ignored.
+
+
+## Processing Algorithm {#processing}
+
+To process a series of bytes to recover the expressed preferences,
+those bytes are parsed into a Dictionary ({{Section 4.2.2 of !FIELDS}}),
+then preferences are assigned to each usage category in the vocabulary.
+
+The parsing algorithm for a Dictionary
+produces a keyed collection of values,
+each with a possibly-empty set of parameters.
+The parsing process guarantees that each key has at most one value and parameters.
+
+To obtain preferences for each of the categories in the vocabulary,
+iterate through the categories.
+For the label that corresponds to that category (see {{t-category-labels}}),
+obtain the corresponding value from the collection,
+disregarding any parameters.
```

Comment:
See - you are ignoring them anyway, so let's disallow from syntax to simplify the parsing rules!